tests/phpunit/tests/user/query.php

```diff
@@ -13,6 +13,8 @@ class Tests_User_Query extends WP_UnitTestCase {
 
 	protected $user_id;
 
+	private $user_query;
+
```

Comment:
As most of the tests in this class are setting query variables for their test, I think this isn't needed. 

Instead, let's instantiate in each new test method.

Reply:
Or maybe other test methods need to be refactored to use the class property instead of local variables.
But I agree that for consistency it's better to use the same approach throughout the test class.